app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PostsController < ApplicationController
+  def id; end
+
```

Comment:
- [ ] When you will open the `/users/:id/posts` route it will look for the `index` action in the posts controller, Rails follows its naming conventions. Please fix this issue by changing the `id` method to `index` and always remember to follow the rails naming conventions.